synapse/models/dns.py

```diff
@@ -123,6 +123,10 @@ def getBaseModels():
                         'doc': 'The requested FQDN'}),
                     ('type', {'ptype': 'inet:dns:type', 'ro': 1, 'req': 1,
                         'doc': 'The type of DNS record requested'}),
+                    ('seen:min', {'ptype': 'time:min',
+                        'doc': 'The earliest observed time for the data in the record.'}),
+                    ('seen:max', {'ptype': 'time:max',
+                        'doc': 'The most recent observed time for the data in the record.'}),
```

Comment:
Since inet:dns:req represents a "request" (an IP address making a DNS lookup) vs. a "record" suggest changing wording: "The most recent observed time that the address made the specified request."

Reply:
made your requested changes